### ISSUE TITLE
JitBench: Failing with missing assembly 'Microsoft.Win32.Registry',

### ISSUE THREAD

**Issue by briansull** [aspnet/JitBench#34]
In the 'dev' branch after following all the steps to build and run:

E:\fxkit\JitBench\src\MusicStore\bin\Release\netcoreapp2.0\publish>dotnet MusicStore.dll
Error: assembly specified in the dependencies manifest was not found probably due to missing runtime store associated with artifact.xml -- package: 'Microsoft.Win32.Registry', version: '4.4.0-preview1-25229-02', path: 'runtimes/win/lib/netcoreapp2.0/Microsoft.Win32.Registry.dll'

**Comment by rynowak** [comment]
Related issue: https://github.com/dotnet/cli/issues/6492

**Comment by rynowak** [comment]
Fixed this issue by just creating the store with the minimal packages used by MusicStore.

**Comment by rynowak** [comment]
This PR has the real workaround https://github.com/aspnet/JitBench/pull/41/files

**Comment by rynowak** [comment]
I'm reactivating this to track it because the workaround results in a few extra binaries that don't get covered by crossgen. 
```
    Directory: C:\Users\rynowak\git\aspnet\JitBench\src\MusicStore\bin\Release\netcoreapp2.0\publish


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
d-----         5/5/2017   9:59 PM                Areas
d-----         5/5/2017   9:59 PM                refs
d-----         5/5/2017   9:59 PM                runtimes
d-----         5/5/2017   9:59 PM                Views
d-----         5/5/2017   9:59 PM                wwwroot
-a----         5/5/2017   1:09 PM            658 config.json
-a----         5/5/2017   6:30 PM          27360 Microsoft.Win32.Registry.dll
-a----         5/5/2017  11:49 PM         231595 MusicStore.deps.json
-a----         5/5/2017  11:49 PM         194048 MusicStore.dll
-a----         5/5/2017  11:49 PM          23460 MusicStore.pdb
-a----         5/5/2017  11:49 PM            240 MusicStore.runtimeconfig.json
-a----         5/5/2017   6:30 PM          40184 System.Security.AccessControl.dll
-a----         5/5/2017   6:30 PM          32008 System.Security.Principal.Windows.dll
-a----         5/5/2017  11:49 PM            473 web.config
```

This will be fixed when we address the issue in the ASP.NET preview-2 packages, then we can remove this workaround

**Comment by rynowak** [comment]
My workaround seems to no longer be working. I've pinged the SDK issue again.

**Comment by rynowak** [comment]
Currently we don't have any workarounds in place for this problem, we're ok. I'm leaving this open for now because the underlying `dotnet store` issue hasn't been fixed and it could pop up again if the packages get out of sync

**Comment by rynowak** [comment]
This should be fixed as of SDK `v2.0.0-preview2-006391`
